### PR TITLE
Add glog recipe

### PR DESCRIPTION
**Pull request by andyneff**
Add a Google Logging recipe to make a glog wheel primarily to add the `installFailureSignalHandler` functionality.

Eventually, probably want to make a separate glog repo, to support Windows, mac, and Linux and auto deploy to pypi. But for now, this will be hosted here for Linux only support.

Also started to update docker files to use `AS` instead of `as` as per new standard.

### CODE REVIEW

File: glog11/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+# Find the module development requirements (requires FindPython from 3.17 or
+# scikit-build-core's built-in backport)
+find_package(Python REQUIRED COMPONENTS Interpreter Development.Module)
+find_package(pybind11 CONFIG REQUIRED)
+# set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "/glog11/cmake")
+# find_package(glog REQUIRED)
+find_package(glog 0.7.1 REQUIRED)
+# find_package(gflags REQUIRED CONFIG)
+
+# Fix for https://github.com/conda-forge/glog-feedstock/issues/26
```
[the remark this answers is not in the document]
Comment (andyneff) [reply glog11/CMakeLists.txt]: iirc updating from 0.7.0 to 0.7.1 fixed it

File: docker-compose.yml [hunk 2]
```diff
@@ -35,3 +34,34 @@ services:
...
+    image: vsiri/blueprint:glog
+
+  # Maintanence
+  glog-compile:
+    image: quay.io/pypa/manylinux_2_28_x86_64
+    command: |
+      bash -euxvc '
+        for v in $$(seq 8 13); do
+          python_version=3"$${v}"
+          python_dir=("/opt/python/cp$${python_version}-"cp*[0-9m]); declare -p python_dir
```
[the remark this answers is not in the document]
Comment (andyneff) [reply docker-compose.yml:57]: I was...

File: blueprint_glog.Dockerfile [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# ARG BASE_IMAGE="quay.io/pypa/manylinux2014_x86_64:2024-07-02-9ac04ee"
+ARG BASE_IMAGE=quay.io/pypa/manylinux_2_28_x86_64
+FROM "${BASE_IMAGE}" AS builder
+
```
[the remark this answers is not in the document]
Comment (andyneff) [reply blueprint_glog.Dockerfile:4]: Yep

File: glog11/CMakeLists.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (scott-vsi) [reply glog11/CMakeLists.txt]: Perhaps a comment to this effect? or take it out?

**Pull request merged**